### PR TITLE
SCUMM: Simplify bug #4556 workaround; whitespace and comment fixes

### PR DESCRIPTION
**Pull request by einstein95**

### CODE REVIEW

File: engines/scumm/script_v2.cpp [hunk 1]
```diff
@@ -1240,7 +1240,7 @@ void ScummEngine_v2::o2_startScript() {
 	// a permanent resident.
 	// Our fix is simply to prevent the Cutscene playing, if the lab has already been stormed
 	if (_game.id == GID_MANIAC) {
-		if script == MM_SCRIPT(150) {
+		if (script == MM_SCRIPT(150)) {
 			if VAR(MM_VALUE(104, 120) == 1)
```
Comment (dwatteau) [inline engines/scumm/script_v2.cpp:1244]: @einstein95 you still have one typo left here

Comment (einstein95) [reply engines/scumm/script_v2.cpp]: Right, I really shouldn't edit using Github's web interface...

**Comment by dwatteau** [comment]
Thanks. I don't see any problem with the proposed changes.

I don't know if removing these empty lines is that useful, though. AFAICS nothing is enforced about this in particular (the rules are here: https://wiki.scummvm.org/index.php/Code_Formatting_Conventions) because it's part of the personal style/preference of each developper. When I add some new code I try to be consistent with surrounding code, and if I see some lines being next to each other with some very different style, I try to harmonize them a bit, but that's it.

**Comment by einstein95** [comment]
Right, I saw code that had no whitespace before it and assumed it to be standard. Would it be better reverting the whitespace changes and leaving the rest?

**Comment by dwatteau** [comment]
> Right, I saw code that had no whitespace before it and assumed it to be standard. Would it be better reverting the whitespace changes and leaving the rest?

Probably, but that's just my opinion so maybe another reviewer would agree with the whitespace changes.

**Comment by einstein95** [comment]
Removed the whitespace changes, as I now reckon they don't need changing.

**Comment by dwatteau** [comment]
Looks all good to me, thanks!

**Pull request merged**